modules/shared/src/main/resources/logback-included.xml

```diff
@@ -110,6 +126,20 @@
             <customFields>{"application":"${application_name}", "peer_id_short": "${self_id}", "ip": "${external_ip}"}</customFields>
         </encoder>
     </appender>
+    <appender name="PEER_SELECT_JSON" class="ch.qos.logback.core.rolling.RollingFileAppender">
```

Comment:
Do you use the json appender? We stream all `.json` logs and I believe peer select is just for debugging.

Reply:
I just copied the log configuration for `Gossip`. What do I change here?

Reply:
Simply remove the json appender

Reply:
The JSON appender has been removed and the configuration updated accordingly.